typescript/infra/scripts/warp-routes/export-warp-configs.ts

```diff
@@ -14,14 +17,22 @@ async function main() {
   for (const warpRouteId of warpIdsToCheck) {
     console.log(`Generating Warp config for ${warpRouteId}`);
 
-    const warpConfig = await getWarpConfig(
+    const warpConfigs = await getWarpConfig(
```

Comment:
it's technically all still part of one `warpConfig` - but with subconfigs per-chain